MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSErrorAttachment.h

```diff
@@ -1,5 +1,7 @@
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I can move the file from `Model` to `MobileCenterCrashes` (physically). Or I can move the file from group `MobileCenterCrashes` to group `Model`.